src/uthash.h

```diff
@@ -119,7 +119,7 @@ do {
 #define HASH_FIND_BYHASHVALUE(hh,head,keyptr,keylen,hashval,out)                 \
 do {                                                                             \
   (out) = NULL;                                                                  \
-  if (head) {                                                                    \
+  if (head != NULL) {                                                            \
```

Comment:
This change is wrong, because `head` is a macro parameter that needs parentheses whenever it's an operand.
(I would welcome a patch that fixed any lingering issues in that area; I'm sure there are some. However, that patch would suffer from the same bit-rot problem as all these "linting" patches: the code would quickly fall out of conformance unless the patch actually added test cases to the test suite exercising the new invariants.)